nvm.sh

```diff
@@ -1828,6 +1828,17 @@ nvm() {
         NVM_PREFIX="$(nvm_node_prefix)"
         NVM_SUCCESS_MSG="Uninstalled node $VERSION"
       fi
+
+      # Check version dir permissions
+      local INVALID_FILES="$(find "$VERSION_PATH" -exec [ ! -w "{}" ] \; -exec echo "{}" \;)"
+      if [ "$INVALID_FILES" ]; then
+        echo "Cannot uninstall, incorrect permissions on installation folder."
+        echo "This is usually caused by running 'npm install -g' as root. Run the following command as root to fix the permissions and then try again."
```

Comment:
let's not double-quote lines that don't use interpolation - also, it'd be nice to put backticks (not single quotes) around the command